### PR TITLE
Show number of selected branding options on Add branding options page

### PR DESCRIPTION
**Pull request by CrystalPea**
Also show a link that allows to clear selection.

Review commit by commit recommended.

### Pivotal ticket:
https://www.pivotaltracker.com/story/show/183013827

### Nothing selected:
Screenshot 2022-09-02 at 16 01 18

### Options selected:
Screenshot 2022-09-02 at 15 15 43

### CODE REVIEW

**Review by tombye — COMMENTED** [review]
I've made a few comments, most of which I think are simple 'we don't need this' things. There's one that might need a bit of discussion and I'll slack you about it.

File: tests/javascripts/addEmailBrandingOptionsForm.test.js [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+});
+
+afterAll(() => {
+  require('./support/teardown.js');
+
+  // tidy up
+  delete window.scrollTo;
+});
+
+describe('TemplateFolderForm', () => {
```
[the remark this answers is not in the document]
Comment (CrystalPea) [reply tests/javascripts/addEmailBrandingOptionsForm.test.js:17]: Nice catch 👏🏼

File: app/assets/javascripts/addEmailBrandingOptionsForm.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+      if (currentStateObj && ('setFocus' in currentStateObj)) {
+        scrollTop = $(window).scrollTop();
+        currentStateObj.setFocus();
+        $(window).scrollTop(scrollTop);
+      }
+    };
+
+    this.nothingSelectedHint = $(`
+      <div id="nothing_selected">
+        <div class="template-list-selected-counter">
```
[the remark this answers is not in the document]
Comment (CrystalPea) [reply app/assets/javascripts/addEmailBrandingOptionsForm.js]: Hey, I did part of this work - the renaming. I did not put the block in a separate file though as it's very small. I hope that's ok.

**Review by tombye — APPROVED** [review]
Looks good to me 👍🏻

**Pull request merged**